src/ludamus/templates/panel/parts/import-recipe-row.html

```
@@ -193,22 +193,32 @@
                        name="droption_{{ row.index }}"
                        value="{{ dur.option }}">
-                <input type="text"
-                       name="driso_{{ row.index }}"
-                       value="{{ dur.iso }}"
-                       aria-label="{% translate "ISO 8601 duration" %}"
-                       placeholder="PT1H30M"
-                       pattern="PT(\d+H)?(\d+M)?"
-                       title="PT1H30M"
-                       class="w-full rounded-lg border border-border bg-bg-secondary px-2 py-1 text-sm text-foreground focus:outline-none focus:ring-2 focus:ring-primary">
+                <div class="flex items-center gap-2">
+                    <input type="number"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)